src/tokenizer/mod.rs

```diff
@@ -270,6 +274,7 @@ impl<Sink: TokenSink> Tokenizer<Sink> {
         if c == '\r' {
             self.ignore_lf = true;
             c = '\n';
+	    self.current_line += 1;
```

Comment:
Let's make a separate block after this instead:
```rust
if c == '\r' {
    self.ignore_lf = true;
    c = '\n';
}

if c == '\n' {
    self.current_line += 1;
}
```
This means that input that only contains \n tokens will correctly count lines, and code that contains \r\n tokens (which is what the ignore_lf checks are for) will also count lines correctly.